docs/README.rst

```diff
@@ -1,49 +1 @@
-###################
-EdX Documentation
-###################
-
-The following documentation projects have been moved to the `edx-documentation`_ repository as of November 3, 2014:
-
-* course_authors
-* data
-* developers
-* install_operations
-* mobile
-* OLX
-* ORA-2
-* release_notes
-* students
-
-API documentation that includes docstrings from code files is stored in the
-repository of that module.  All other documentation projects are stored in edx-
-documentation.
-
-By moving documentation to its own repository, we will be better able to
-develop workflows, manage versioning, create translations, and automate
-testing, without complicating ongoing development of the edX Platform.
-
-.. _edx_documentation: https://github.com/edx/edx-documentation
-
-******************************
-View Published Documentation
-******************************
-
-EdX documentation is published through Read the Docs. Links to all published
-documentation are available through `docs.edx.org`_.
-
-.. _docs.edx.org: http://docs.edx.org
-
-******************************
-Submit Documentation Issues
-******************************
-
-We welcome input from the community on any documentation issues.  You can
-submit issues to the Documentation project in the `Open edX JIRA board`_.
-
-You will need to `create a free JIRA account`_ account before you can submit your first
-ticket.
-
-.. _create a free JIRA account: https://openedx.atlassian.net/admin/users/sign-up
-.. _Open edX JIRA board: https://openedx.atlassian.net
-
-You can also email docs@edx.org.
+See `index.rst <index.rst>`_ for details.
```

Comment:
REVIEWERS: The docs team seems to separate into source/build directories.  The IDA cookiecutter just has the docs in the root, as well as the config file for building, etc.  I decided to start simple and we can change if necessary when building.